Dockerfile.aro-multistage

```diff
@@ -1,7 +1,7 @@
 # Uses a multi-stage container build to build the RP.
 #
 ARG REGISTRY
-FROM ${REGISTRY}/ubi8/go-toolset:1.21.13-1 AS builder
+FROM ${REGISTRY}/ubi8/go-toolset:1.22.9-2 AS builder
```

Comment:
Can we also get a PR out to ARO-Build-Images as well? We'll need it to publish this image to our isolated ACR. Ref: https://msazure.visualstudio.com/AzureRedHatOpenShift/_wiki/wikis/ARO.wiki/189194/CDPx-Container-Images?anchor=adding-new-images-to-the-isolated-acr

Reply:
Yep sorry, forgot to link it, but already done: https://msazure.visualstudio.com/DefaultCollection/AzureRedHatOpenShift/_git/ARO.Build.Images/pullrequest/11650462

Reply:
Yep sorry, forgot to link it, but already done: https://msazure.visualstudio.com/DefaultCollection/AzureRedHatOpenShift/_git/ARO.Build.Images/pullrequest/11650462